package.json

```diff
@@ -8,7 +8,9 @@
     "test": "test"
   },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "gulp test",
+    "preinstall": "npm install -g typings",
```

Comment:
@itsananderson - This wasn't working for me locally.  I'll work with you and take this as a seperate PR.